agent/provision_agent.go

```diff
@@ -172,3 +175,18 @@ func (a *provisionAgent) deleteContainer(sc pfmodel.Container, lcs *pfmodel.Cont
 
 	return true, nil
 }
+
+func (a *provisionAgent) bootstrapProcess() bool {
+	log.WithFields(log.Fields{}).Warn("Starting bootstrap agent...")
```

Comment:
Reword this to `Bootstrapping...`
we don't actually start another agent because it runs in the same goroutine